Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrDriver.kt

```diff
@@ -35,6 +35,7 @@ class StubIrContext(
                     when (configuration.library.language) {
                         Language.C -> emptyList()
                         Language.OBJECTIVE_C -> listOf("void objc_terminate();")
+                        Language.J2ObjC -> emptyList()
```

Comment:
Language.J2ObjC -> fail("Unreachable code")

This code in generally looks a little convoluted.

Reply:
Done.

> This code in generally looks a little convoluted.

Can you explain more on this?

Reply:
The code is:

if (object-c) {
  libraryForCStubs = callfunction
} {
 libraryForCStubs = copy ...
}

The break down of the code here is odd and likely needs some comment to stay this way (and is a little to complex for a ternary expression)

Maybe a when expresion would work better.

Or maybe a multi-line ternary expression.

Reply:
Ok, made it a when expression.